modules/distr-tracing-otel-config-collector.adoc

```diff
@@ -239,6 +239,57 @@ The Zipkin receiver ingests data in the Zipkin v1 and v2 formats.
 [id="processors_{context}"]
 === Processors
 
+
+[id="batch-processor_{context}"]
+==== Batch processor
+
+The batch processor batches the data to reduce the number of outgoing connections needed to transfer the telemetry information.
+
+* Support level: link:https://access.redhat.com/support/offerings/techpreview[Technology Preview]
+* Supported signals: traces, metrics
+
+.Example of the OpenTelemetry Collector custom resource when using the Batch processor
+[source,yaml]
+----
+  config: |
+    processor:
+      batch:
+        timeout: 5s
+        send_batch_max_size: 10000
+    service:
+      pipelines:
+        traces:
+          processors: [batch]
+        metrics:
+          processors: [batch]
+----
+
+.Parameters used by the Batch processor
```

Comment:
Looks like it's a lowercase "batch processor":
https://opentelemetry.io/search/?q=batch%20processor
https://github.com/open-telemetry/opentelemetry-collector/tree/main/processor/batchprocessor
```suggestion
.Parameters used by the batch processor
```